src/lib/data/near.js

```diff
@@ -292,9 +292,9 @@ async function signCalimeroFakTransaction(storageKey, near, contractName, method
       contractName, [
         nearAPI.transactions.functionCall(
           methodName,
-          {...args},
-          gas?.toFixed(0),
-          deposit?.toFixed(0)
+          args,
+          gas ?? TGas.mul(30).toFixed(0),
```

Comment:
Should we max it out to 300TGas?

https://github.com/near/near-api-js/blob/53ba3f21c6da503e971a251b205bf50ea8b36dd0/packages/utils/src/constants.ts#L9

Reply:
I used the one provided in the VM but yeah, I agree, it's free gas so